tests/4.5/target/test_target_is_device_ptr.c

```diff
@@ -1,52 +1,54 @@
+//===--- test_target_is_device_ptr.c-----------------------===//
```

Comment:
(coding style fix) Top line of header is short, should match length of bottom line.